src/backgroundblurprocessor/BackgroundBlurProcessorProvided.ts

```diff
@@ -47,6 +47,20 @@ export default class BackgroundBlurProcessorProvided
     this.logger.info('BackgroundBlur processor successfully created');
     this.logger.info(`BackgroundBlur spec: ${this.stringify(this.spec)}`);
     this.logger.info(`BackgroundBlur options: ${this.stringify(options)}`);
+
+    screen.orientation.addEventListener('change', _event => {
+      if (
+        (screen.orientation.type.startsWith('portrait') &&
+          this.canvasCtx.canvas.height < this.canvasCtx.canvas.width) ||
```

Comment:
Can you put a comment that explains what this is doing in plain english? It's a bit tricky at first glance even if its doing something simple.

Reply:
Sure, I can add a comment